pkg/resource/model_package/testdata/v1alpha1/create/desired/success_before_create.yaml

```diff
@@ -0,0 +1,42 @@
+apiVersion: sagemaker.services.k8s.aws/v1alpha1
```

Comment:
nit: not sure what the filename means - success before create for a desired spec ?

Reply:
miss on my part renamed to before_create